src/main/java/com/hartwig/platinum/ApiRerun.java

```diff
@@ -33,12 +39,36 @@ public ApiRerun(final RunApi runApi, final SetApi setApi, final SampleApi sample
     }
 
     public long create(final String tumorSampleName) {
-        return sampleApi.list(null, null, null, null, SampleType.TUMOR, tumorSampleName, null)
-                .stream()
-                .findFirst()
-                .flatMap(sample -> OnlyOne.ofNullable(setApi.list(null, sample.getId(), true), SampleSet.class))
```

Comment:
As discussed we can simplify this somewhat by removing the `OnlyOne` and instead locating the active set here in a flatMap either by:
- Modifying the API to return the active set
- Doing a filter for Validated runs per set here